.github/workflows/pr-closed.yml

```diff
@@ -0,0 +1,42 @@
+name: PR Closed Cleanup
+# Remove prereleases that were part of a closed PR, so we save space
+# on our deployment targets. If needed, one can still get the
+# spack.yaml as part of the closed PR and revive it themselves.
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+      - backport/*.*
+    paths:
+      - config/**
+      - spack.yaml
```

Comment:
Do we need/want `paths` in here? What conditions would need to be met for us to be sure there had been a prerelease environment deployed?